Sources/OktaAnalytics/OktaAnalytics.swift

```diff
@@ -37,6 +40,18 @@ public final class OktaAnalytics: NSObject {
         return AnalyticsStorage(logger: logger)
     }()
 
+    /**
+     Adds provider to collection
+
+     - Parameters:
+        - securityAppGroupIdentifier: shared group Identifier to be data shared between app, extensions, app and widgets.
+        - expirationPeriodForScenarioEvents: expiration period for scenario events.
+     */
+    public static func initializeStorageWith(securityAppGroupIdentifier: String, expirationPeriodForScenarioEvents: UInt) {
```

Comment:
We don't need 2 funtions @artemshvydkyi-okta , we can use existing `initializeStorageWith ` and pass default time.